### ISSUE TITLE
Not possible to add posts/documents with dates in the future

### ISSUE THREAD

**Issue by musicEnfanthen** [music-encoding/music-encoding.github.io#162]
Per default, Jekyll excludes documents and posts from the build that have a future date in their front matter (due to its blog system background). This way, e.g. upcoming conferences cannot be announced.

**Comment by musicEnfanthen** [comment]
One possible solution is to include

`future: true`

in the `_config.yml`.

**Comment by craigsapp** [comment]
The reason for this, at least for "posts", is that you can prepare news to be released in the future (while you are on vacation :-).

**Comment by musicEnfanthen** [comment]
Yes, I see. Without this background, it was quite troublesome to find out why the MEC2021 files would not show up after building. It is fixed now in eaeb9ee6af7824d58e93b3b9e9eb3ca6432ea20f.

Just wanted to document it for "future" ;)

**Comment by musicEnfanthen** [comment]
@bwbohl Maybe another option would be to use `date-start` in the front matter of the conference index.html instead of `date`? With a global `future: true`, we will not be able to prepare posts to be released in the future as mentioned by @craigsapp above.

**Comment by kepper** [comment]
I would also prefer to keep that option. However, we should make it clear that `date` means the date of publication, not the `date` of an event…

**Comment by musicEnfanthen** [comment]
Ok, I tested with `date-start` option and everything works as expected (future posts are not displayed, but future conferences are). Will make another PR to implement this after #163 has been merged.

Thanks to all.

**Comment by bwbohl** [comment]
actually we could go with `future:true` if we use the `published:false|true` option in the YAML header, e.g. in https://github.com/music-encoding/music-encoding.github.io/blob/master/_conferences/2020/index-old.html

**Comment by musicEnfanthen** [comment]
Probably, leaving the default (future:false) is more intuitive.
